arcsi/static/component.json

```diff
@@ -189,7 +189,7 @@
                     }
                 }
             },
-            "ShowRequest": {
+            "ShowSubPageRequest": {
```

Comment:
We used the docs today and it was very clean and easy to browse. Thanks a lot for building this library of knowledge!!!

Reply:
My pleasure!